cpp/src/slice2js/Gen.cpp

```diff
@@ -315,8 +342,6 @@ namespace
         const OperationPtr& op,
         const CommentPtr& doc,
         OpDocParamType type,
-        bool showExceptions,
```

Comment:
Removed, we should always write the @throws clauses. We were passing "false" all the time and exceptions were ignored.